### PR TITLE
W-10588797-xml-module-troubleshooting-kt

### PR DESCRIPTION
**Pull request by kevintroller**
W-10588797-xml-module-troubleshooting-kt

### CODE REVIEW

File: xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+To troubleshoot XML module, become familiar with the information about enabling verbose logging and interpreting commonly thrown messages.
+
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with the XML Module is failing, temporarily enable verbose logging for XML module. +
+
+Remember to always disable the enhanced verbosity after troubleshooting because it affects your Mule application performance.
+
+To enable verbose logging in the configuration file:
```
Comment (dhirsch1) [inline xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc]: Is there another way to enable verbose logging for this module? If not, I suggest "To enable verbose logging:"

File: xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+[source,xml,linenums]
+----
+<xml-module:config name="XML_Config" expandEntities="INTERNAL"/>
+----
+
+
+== Understand Common Throws
+
+Here is a list of common throw messages and how to interpret them:
```
Comment (dhirsch1) [inline xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc:74]: I would alphabetize these.

File: xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+ The supplied schema is invalid.
+
+* XML-MODULE:SCHEMA_NOT_FOUND
+
+ The schema could not be found.
+
+* XML-MODULE:SCHEMA_NOT_HONOURED
+
+ The input XML document does not honour its schema.
```
Comment (dhirsch1) [inline xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc:90]:
```suggestion
 The input XML document does not adhere to its schema.
```

Comment (dhirsch1) [inline xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc]: "does not adhere to its schema" 

If you want to keep "honour," change it to "honor," which is the American spelling.

File: xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with the XML Module is failing, temporarily enable verbose logging for XML module. +
+
+Remember to always disable the enhanced verbosity after troubleshooting because it affects your Mule application performance.
+
+To enable verbose logging in the configuration file:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
```
Comment (dhirsch1) [inline xml/1.3/modules/ROOT/pages/xml-module-troubleshooting.adoc:14]: You use "app" instead of "application later on.

**Review by dhirsch1 — DISMISSED** [review]

File: xml/1.3/modules/ROOT/nav.adoc [hunk 5]
```diff
@@ -7,3 +7,4 @@
 ** xref:xml-xpath.adoc[Configure XPath Extractions]
 ** xref:xml-xslt.adoc[Configure XSLT Transformations]
 ** xref:xml-schema-validation.adoc[Validate Documents against an XSD Schema]
+* xref:xml-moddule-troubleshooting.adoc[Troubleshooting XML Module]
```
Comment (sathya0) [inline xml/1.3/modules/ROOT/nav.adoc]:
```suggestion
* xref:xml-module-troubleshooting.adoc[Troubleshooting XML Module]
```

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — APPROVED** [review]

**Pull request merged**